definitions/infra-container/summary_metrics.yml

```diff
@@ -8,15 +8,15 @@ cpuUsageContainer:
   query:
     eventId: entityGuid
     from: ContainerSample
-    select: average(cpuUsedCores)
+    select: max(cpuUsedCores)
```

Comment:
are we sure about this `max` ?

Reply:
Yes, as it's the same value function that is used for the corresponding GM.